src/sa_web/jstemplates/place-form.html

```diff
@@ -16,101 +16,125 @@ <h4 class="">{{ place_config.title }}</h4>
          is already logged in.
```

Comment:
It doesn't seem like our dynamic form is using placeFormView or this placeForm template, so I don't see why we still have changes here. For example, you can undo the changes in this file and it shouldn't have any effect on the dynamic form.

We may want to consider using a single view/template for all of our place forms, which basically means that we'd delete the dataset view/template and integrate them back into our placeForm view/template. Sorry for the confusion, because I remember that this is how you initially had it working. We can discuss this in person, but it seems most reasonable to use a single view to handle our place forms.